docs/transcript_formats.md

```diff
@@ -51,9 +52,39 @@ A middle ground between `format: raw` and `format: timestamped-words`. Like all,
 happen on a sentence level), ten `data` portion is a large list where each `text` portion of each dictionary in the list
 is a single sentence (including any punctuation returned from the model).
 
+### Format: Timestamped Speaker Turns
+An enhancement of `format: timestamped-sentences`. Like all, it has the basic `format`, `annotations`, and `confidence`.
+However, the `data` section is a large list of speaker turns, where each speaker turn is a dictionary with keys: `speaker`
+and `data`. `speaker` is a text representing the identity of the speaker. `data` is a list of sentences belonging to
+a particular speaker turn. These sentences have the same structure as the elements of `data` of `format: timestamped-sentences`.
+
+The basic layout of a `format: timestamped-speaker-turns` format currently for CDP instances will be the following JSON block:
+
+```json
+{
+    "format": "timestamped-speaker-turns",
+    "annotations": [
+        {}
+    ],
+    "confidence": 0.0,
+    "data": [
+        {
+            "speaker": "speaker-0",
```

Comment:
I would add a comment that you can leave this as an empty string if it is unknown.